src/manager/agent.config.ts

```diff
@@ -133,7 +139,29 @@ export class AgentConfig {
             '--webhook-url', this.webhookUrl,
             '--log-level', this.logLevel,
             '--wallet-local-did', // TODO this could be an arg on the config
-        ];
+            '--tails-server-base-url', process.env.TAILS_URL,
+            );
+        } else {
```

Comment:
Instead of copy-pasting this block, you can just do what we do with if (this.seed) below and just args.push the additional param